bumble/snoop.py

```diff
@@ -85,7 +85,13 @@ def snoop(self, hci_packet: bytes, direction: Snooper.Direction) -> None:
 
         # Compute the current timestamp
         timestamp = (
-            int((datetime.datetime.utcnow() - self.TIMESTAMP_ANCHOR) / self.ONE_MS)
+            int(
+                (
+                    datetime.datetime.now(tz=datetime.timezone.utc)
+                    - self.TIMESTAMP_ANCHOR
```

Comment:
I think that `TIMESTAMP_ANCHOR` would need to be updated as well, to be a timestamp *with* an explicit UTC timezone.